src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -75,6 +75,8 @@ public abstract partial class Frame : IFrameControl
         protected readonly long _keepAliveMilliseconds;
         private readonly long _requestHeadersTimeoutMilliseconds;
 
+        private int _responseBytesWritten;
```

Comment:
This should be long. Maybe call this _responseBytesIgnored since we're only using it to log the number of bytes not written.